src/main/java/step4/dto/car/CarGroup.java

```diff
@@ -0,0 +1,56 @@
+package step4.dto.car;
+
+import step4.dto.count.Position;
+import step4.dto.name.CarNameGroup;
+import step4.dto.record.Record;
+import step4.dto.count.RandomNumber;
+import step4.dto.record.RecordGroup;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+public class CarGroup {
+    List<Car> cars;
+
+    public CarGroup(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void run() {
+        cars.stream()
+                .forEach(car -> car.run(RandomNumber.newRandom()));
```

Comment:
List 자료구조는 stream을 사용하지 않아도 자체적으로 forEach 메서드를 지원합니다. 
```suggestion
        cars.forEach(car -> car.run(RandomNumber.newRandom()));
```